packages/node_modules/@webex/plugin-logger/src/logger.js

```diff
@@ -58,8 +62,11 @@ function walkAndFilter(object, visited = []) {
   if (!isObject(object)) {
     if (isString(object)) {
       if (patterns.email.test(object)) {
-        return '-- REDACTED --';
+        return object.replace(patterns.email, REDACTED);
       }
+      // if (patterns.email.test(object)) {
+      //   return '-- REDACTED --';
+      // }
```

Comment:
I'm guessing you didn't mean to leave this in?

Reply:
Removed this.